web/src/compose_closed_ui.ts

```diff
@@ -83,9 +90,36 @@ function update_reply_button_state(disable = false): void {
 }
 
 function update_buttons(disable_reply?: boolean): void {
+    if (disable_reply && maybe_get_selected_message_stream_id()) {
+        $("#new_conversation_button").prop("disabled", true);
+    } else {
+        $("#new_conversation_button").prop("disabled", false);
+    }
     update_reply_button_state(disable_reply);
 }
 
+function maybe_get_selected_message_stream_id(): number | undefined {
+    if (message_lists.current?.visibly_empty()) {
+        return undefined;
+    }
+    const selected_message = message_lists.current?.selected_message();
+    if (!selected_message?.is_stream) {
+        return undefined;
+    }
```

Comment:
This section won't be needed since if message is private then `message.stream_id` anyways returns undefined